distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,845 @@
+
+|id
+|ID of the activity
+|String
+|Any String
+|"a62f6666-fc41-4a19-91f1-485e73a564b5"
+|The Activity ID to which the requested operation relates
+|===
+
+.Example: Activity Operation Request Message
+[source,json,linenums]
+----
+"data": [ {
+	"action":"cancel",
+ 	"id":"a62f6666-fc41-4a19-91f1-485e73a564b5"
+} ]
+----
+
+====== Query Service Channel
+
+All query requests should be published to the `/service/query` channel.
+
+When performing a CometD publish command, the data being published must be valid json with 'data' being the key to a map that contains the following values:
+
+.Query Request Format
+[cols="1m,3,1,1,1,1" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Search UI Values
+|Example Values
+|Comments
+
+|count
+|Number of entries to return in the response
+|Number
+|Positive integer
+|250
+|
+
+|format
+|Format that the results should be displayed in
+|String
+|"geojson"
+|"geojson"
+|"geojson" is the recommended format to use
+
+|id
+|
+|String
+|
+|"4303ba5d-21af-4878-9a4c-808e80052e6c"
+|
+
+|src
+|Comma-delimited list of federated sites to search over
+|String
+|Any
+|list of site names.
+|"${ddf-branding}-OS,${ddf-branding-lowercase}.distribution"
+
+|start
+|Specifies the number of the first result that should be returned
+|Number
+|Positive integer
+|1
+10 would mean the 10th result from the query would be returned as the first one in the response.
+
+|cql
+|Search Filter
+|String
+|OGC CQL formatted string
+|"anyText LIKE '*'"
+|See http://www.opengeospatial.org/standards/cat[OpenGIS® Catalogue Services Specification] for more details.
+|===
+
+===== Query Request Examples
+
+.Enterprise Contextual Query
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+ 	"format": "geojson",
+ 	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+ 	"cql": "anyText LIKE '*'",
+ 	"src": "${ddf-branding}-OS,${ddf-branding-lowercase}.distribution",
+ 	"start": 1
+}
+----
+
+.Multiple Site Temporal Absolute Query
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+	"format": "geojson",
+	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+	"cql": "modified DURING 2014-09-01T00:00:00Z/2014-09-30T00:00:00Z",
+ 	"src": "${ddf-branding}-OS,${ddf-branding-lowercase}.distribution",
+ 	"start": 1,
+}
+----
+
+.Enterprise Spatial Bounding Box Query
+[source,json,linenums]
+----
+"data": {
+	"count": 250,
+	"format": "geojson",
+	"id": "4303ba5d-21af-4878-9a4c-808e80052e6c",
+	"cql": "INTERSECTS(anyGeo, POLYGON ((-112.7786 32.2159, -112.7786 45.6441, -83.7297 45.6441, -83.7297 32.2159, -112.7786 32.2159)))",
+	"start": 1
+}
+----
+
+====== Query Response Channel
+
+The query responses are returned on the `/<id>` channel, which should be subscribed to in order to retrieve the results.
+Replace `<id>` with the id that was used in the request.
+The <<<<asynchronous_search_subscribing, Subscribing to Notifications>>>> section details how to subscribe to a CometD channel.
```

Comment:
Do you need all the `<<<` and `>>>`? It seems to render an extra two of each in the PDF at least.